bins/revme/Cargo.toml

```diff
@@ -16,17 +16,20 @@ indicatif = "0.17"
 plain_hasher = "0.2"
 revm = { path = "../../crates/revm", version = "3.5.0", default-features = false, features = [
     "ethersdb",
-    "std",
     "serde",
     "c-kzg",
```

Comment:
`c-kzg` is hard to build in no_std environment, it is `c` lib. But c-kzg is needed for the test so it needs to be included.

I am fine with making `revme` std only as it is binary and more like a tool/utility.

Reply:
Maybe it can be a test only / bench only dependencies?

Reply:
I looked again, and it does make sense to keep this on std cuz it's a command-line app. I'm gonna revert changes for `revme`